src/Helpers/babylon.videoDome.ts

```diff
@@ -0,0 +1,58 @@
+module BABYLON {
+    /**
+     * Display a 360 degree video on an approximately spherical surface, useful for VR applications or skyboxes, video attempts to autoplay.
+     * Subclass of Node so there can be multiple locational videos or instead have it parented to the camera.
+     * This effect can be achieved by using a VideoTexture and correctly configured material on an inverted sphere.
+     * Potential additions to this helper include zoom and and non-infinite distance rendering effects.
+     */
+    export class VideoDome extends Node {
+
+        /**
+         * The video texture being displayed on the sphere
+         */
+        private _videoTexture: VideoTexture;
+
+        /**
+         * The skybox material
+         */
+        private _material: StandardMaterial;
+
+        /**
+         * The surface used for the skybox
+         */
+        private _mesh: Mesh;
+
+        constructor(name: string, urlsOrVideo: string[] | HTMLVideoElement, options: {

```

Comment:
Constructor needs comment

Reply:
Ran into several issues with the documentation compiler trying to write the comments I wanted to write, I'm current examining the documentation process to see if it's me or it has a couple of bugs (seems probable form the output)

Reply:
Cc @sebavan